README.md

```diff
@@ -1,57 +1,52 @@
-## Faros CI/CD GitHub Action
+# Faros CI/CD GitHub Action
 
-A GitHub Action to report CI/CD information for builds and deployments from a GitHub Action workflows to [Faros](https://www.faros.ai).
+A GitHub Action to report CI/CD information for code builds and deployments from a GitHub Action workflows to [Faros](https://www.faros.ai).
 
 ## Usage
 
 See [action.yml](action.yml) for the full documentation for this action's inputs and outputs.
 
-### Report a Build Event To Faros CI/CD Build Model
+### Report a CI Event To Faros
 
-To report a build event to Faros specify build in the `model` parameter and the build details.
+To report a code build to Faros specify `CI` in the `event` parameter and include the `CI` required fields.
 
 ```yaml
-- name: Report build info to Faros
-  id: emit-build-info
+- name: Report code build to Faros
+  id: send-ci-event
   uses: faros-ai/faros-cicd-github-action@main
   with:
     api-key: ${{ secrets.FAROS_API_KEY }}
     api-url: ${{ env.FAROS_API_URL }}
-    model: build
-    build-pipeline-id: build-deploy-workflow # unique id to identify the workflow running the build
-    status: Success                          # possible values - Success, Failure, Cancelled otherwise defaults to Custom
-    started-at: 1594938057000
-    ended-at: 1594948069000
+    event: CI
+    artifact: Docker://my-org/my-repo/artifact-id
+    run-status: Success               # possible values - Success, Failure, Cancelled otherwise defaults to Custom
+    run-started-at: 1594938057000
+    run-ended-at: 1594948069000
 ```
 
-> :clipboard: Note: Whilst the `build-pipeline-id` is optional, it is recommended to provide one to uniquely identify this workflow in Faros since multiple GitHub workflows can have the same name. If the `build-pipeline-id` is not provided it will be generated from the workflow name in the lowercase format `GITHUB_ORG/REPO/GITHUB_WORKFLOW_NAME`, e.g `my-org/my-repo/deployment`.
+### Report a CD Event To Faros
```

Comment:
why not `Report a Deployment (CD) Event`, same for the `CI` Event above. Prefer not to have abbrv